### ISSUE TITLE
Dsurround and Future Pack compatibility issue

### ISSUE THREAD

**Issue by KangyKSLG** [OreCruncher/DynamicSurroundings#600]
**Mod Version:**

3.6.1.0

**Forge Version:**

forge-14.23.5.2854

**Link to client log:**

Latest log attached as file, log too big for pastebin and hastebin
[latest.log](https://github.com/OreCruncher/DynamicSurroundings/files/5434552/latest.log)

**Link to crash log:**

Crash log https://pastebin.com/YAcMYNSE

**Description:**

Description of the problem.  In the case of crashes, it is good to know things like the following:

* Did you just add the mod to a pack? No, this was already in the pack. I believe it may be compat issues with FuturePack as seen previously in #550 only reporting here first as the log points to dsurround
* Did it just start crashing, or has it always crashed? No
* Did you update any other mods in your pack and it started crashing?  Added Future Pack
* If updating multiple mods did you update one by one or all at once?  N/A
* Is the modpack you are using available online somewhere?  Custom modpack i'm developing to release on CurseForge

Crashes in the final initialisation stages just before the main menu would be shown

**Comment by OreCruncher** [comment]
Try 3.6.1.1.  You will need to install MixinBootstrap as well (current version - the mod works on multiple different versions of MC).

**Comment by KangyKSLG** [comment]
Thanks for the response, I can only find a 3.6.2.0 beta version. Is this the one you meant?

**Comment by KangyKSLG** [comment]
I tried with 3.6.2.0 and the crash still happens
[crash-2020-10-25_19.29.55-client.txt](https://github.com/OreCruncher/DynamicSurroundings/files/5435793/crash-2020-10-25_19.29.55-client.txt)
[latest.log](https://github.com/OreCruncher/DynamicSurroundings/files/5435794/latest.log)

**Comment by OreCruncher** [comment]
Is this the first time adding DS to your pack?  It's crashing because DS cannot find one of its own classes, meaning that the mod didn't load all the way, or something prevented some of its classes from loading.  Can you disable VanillaFix to see what the callstack is without it's involvement?

**Comment by KangyKSLG** [comment]
No, DS has been in the pack since day 1. Here are the logs with VanillaFix disabled. Let me know if you need anything else from me :)
[latest.log](https://github.com/OreCruncher/DynamicSurroundings/files/5437691/latest.log)
[crash-2020-10-26_08.34.40-client.txt](https://github.com/OreCruncher/DynamicSurroundings/files/5437692/crash-2020-10-26_08.34.40-client.txt)

**Comment by OreCruncher** [comment]
So what changed from the time DS was working until it wasn't.  Did you just upgrade DS, or were other mods added/removed/upgraded as well?

**Comment by KangyKSLG** [comment]
The only change is the addition of future pack. If I disable it, DS loads. 
Like I said earlier,  reporting here first due to the logs pointing your way.  It could well be future packs fault

**Comment by OreCruncher** [comment]
Found this gem: https://github.com/MinecraftForge/MinecraftForge/issues/6719.  Its for 1.15.x, but I believe it is in 1.12.x as well.

**Comment by KangyKSLG** [comment]
Interesting. So this could be an inherent Forge issue. Does that mean currently there is no way to solve this and I'll need to choose between future pack and DS?

If that's the case I'll have to live without future pack as DS is too good to exclude

**Comment by OreCruncher** [comment]
It looks to be a Forge issue.  And unfortunately it would require a choice on your part.  I will keep mulling things over to see if I can figure a way around this issue.

**Comment by KangyKSLG** [comment]
No worries, thank you for taking the time to look at this. Its just weird it always seems to be DS and Future Pack that cause it haha
